### DISCUSSION TITLE
Rendere Smart Citofono Urmet 1130 con Shelly 1 v3 e Relè TWTADE AC 12V 8 pin DPDT (2NO 2NC)

### DISCUSSION THREAD

**Discussion by ledwallet77** [indomus/forum#1104] [GENERALE - Discussioni generiche]
Buonasera a tutti,

                           sto cercando di mettere in pratica prendendo spunto da varie fonti in rete il mio vecchio citofono e sono arrivato a definire questo schema:

![schema-1](https://user-images.githubusercontent.com/94041891/143314586-35b2ec1c-6909-49dc-b60e-4e0a3f68bc2c.png)

Purtroppo non riesco a far funzionare il relè, in quanto, non sono pratico di elettricità epotrei aver sbagliato i collegamenti, mi può aiutare qualcuno?

Grazie in anticipo

**Comment by Larry-78** [comment]
Ciao,
se non sbaglio avevi già aperto una discussione sullo stesso argomento, dalle foto confermo che il tuo citofono è un URMET 1130 (2011 lo hai letto nella parte interna ma non è il codice esatto) e quando chiedevi quale dispositivo utilizzare, la risposta è qualsiasi dispositivo, l'importante è che abbia un contatto pulito. Noto che in questa seconda discussione hai uno Shelly 1 v3 e va benissimo (lo puoi alimentare anche a 12V DC cambiando un jumper), non so dove hai recuperato quello schema ma la cosa è più semplice di quanto sembra. Ti do qualche consiglio in seguito alla mia esperienza. 
Prima cosa: una domanda stupida, il citofono funziona? Se lo hai smontato per fare le prove oppure hai collegato relè o shelly, togli tutto e ricollega i fili della montante citofonica nel modo corretto, chiudi la copertura che mantiene la cornetta e assicurati il funzionamento (parlato/ascolto/apertura) dopodiché apri la copertura e devi fare una semplice prova, procurati un pezzettino di filo per creare un ponticello e devi chiudere il 6 con il 9, se il portone/cancello si apre BENE, ti basta collegare i contatti O - I dello Shelly sul 6 e 9 come nello schema che hai condiviso. Devi fare attenzione! devi impostare lo Shelly ad impulso e non interruttore, se non sbaglio si può cambiare nelle impostazioni, non ho quel dispositivo quindi non so di preciso ma è importante non lasciare chiusi 6 e 9 per molto tempo altrimenti si potrebbe bruciare l'elettro-serratura. In questo modo potrai attraverso lo Shelly comandare la chiusura del suo relè interno e di conseguenza l'apertura del portone, basterebbe integrarlo in HA per creare automazioni secondo le tue esigenze.

**Reply by ledwallet77** [reply]
Buongiorno Larry,
                           non sbagli, avevo iniziato una discussione mi pare per chiedere consiglio tra sue dispositivi sonoff e shelly.

Intanto ti ringrazio per la risposta, mi fà molto piacere, senza condivisione ed aiuto non si và da nessuna parte.

Detto ciò voglio provare a risponderti partendo dallo schema: l'ho reperito da un blog con relativo video su youtube in cui originariamente era presente un regolatore di tensione tra il citofono e il relè(che era un altro tipo non quello attuale sullo schema).

In base ai commenti sul blog e sul video, alcuni utenti come stai facendo tu, hanno fatto notare alcune cose una tra le quali che il regolatore di tensione non era necessario, infatti ho modificato lo schema ed ho eliminato il regolatore di tensione.

Sto postando l'immagine del citofono con i collegamenti(credo che il ponte tra 6-9 ci fosse già, io non l'ho fatto cmq e non si è bruciato nulla per fortuna): alimentato con presa, attualmente ho testato, purtroppo sono solo, che il citofono riceve audio(lo sento dalla cornetta), ho registrato con smartphone che dal pulsante in strada il ronzatore funziona ed anche con app shelly premendo da app il tasto apertura(con autospegnimento a 1 sec.) il citofono apre in remoto.

Quello che a me serve, e che non ho capito se funziona o no il relè nel senso che l'unica parte in cui non sono sicuro sono i collegamenti dal/al citofono e viceversa al/dal relè, è che tramite app di shelly possa fare le seguenti operazioni:

- rendere tramite la pressione del tasto della pulsantiera da strada verso il citofono l'apertura automatica con un lasso di tempo di 10 secondi per esempio;
- temporizzare l'apertura automatica appena descritta, in una fascia oraria per esempio dalle 16:00 alle 19:00.

Nel ringraziarti ancora, spero che tu mi possa dare una mano, perchè sono bloccato ........

![urmet](https://user-images.githubusercontent.com/94041891/143410554-f1a16ec9-96b3-4a84-a23f-449bc25e57b4.JPG)

**Reply by Larry-78** [reply]
È un piacere se posso esserti di aiuto. Mi sembra strano che tu abbia trovato un ponticello su 6 e 9, io te l'ho proposto solo per fare un test apertura chiudendo i contatti per pochi secondi, ad ogni modo non entrando in merito ai collegamenti, hai detto che comunque riesci ad aprire da remoto attivando Shelly e questo basta. 
Come dicevo mi affido sulle mie esperienze e anche io ho domotizzato il citofono, con un dispositivo diverso ma il concetto è simile, per me va bene così perché ho integrato lo switch in HA e ho aggiunto un widget nell' app companion quindi ho un bel bottone nell' home del mio smartphone con il quale apro il portone, visto che ormai viviamo con lo smartphone in mano, lo trovo molto comodo e rapido. Entrando in merito alla tua richiesta se non ho capito male vorresti attivare lo Shelly con un ritardo di 10 sec attraverso la pressione del tasto del posto esterno e all'interno di una fascia temporale 16 - 19. Si potrebbe fare ma dovresti aggiungere un altro dispositivo che rilevi la tensione presente sul contatto 7 e attivi un relè che chiuda i contatti 6 - 9 tutto questo potrebbe ripercuotersi sulla montante citofonica e creare disturbi. Io trovo più pratica e meno invasiva la mia soluzione premendo un bottone sullo smartphone, invece di premere il tasto del posto esterno, poi ognuno è libero di seguire le proprie idee e scelte in base alle proprie esigenze, il mio è soltanto un parere.
